assets/js/quizgame.js

```diff
@@ -0,0 +1,287 @@
+    answerButton.setAttribute("class", "answers-btn");
+    answerButton.setAttribute("id", index);
+    answerButton.setAttribute("data-option", answer);
+    answerButton.textContent = answer;
+    questionsDiv.appendChild(answerButton);
+  };
+
+  // For each answer, construct an answerButton along with Class, ID and Data-Attribute.
+  currentQuestion.answers.forEach(renderAnswers);
+};
+
+// Once the answer has been selected, validate
+const answerValidation = function (event) {
+  const currentTarget = event.currentTarget;
+  const target = event.target;
+
+  const correctAnswer = currentTarget.getAttribute("data-answer");
+  const userAnswer = target.getAttribute("data-option");
+
+  // Initial IF statement to ensure the game isn't over, Second IF to carry out game logic
+  if (currentIndex < myQuestions.length - 1) {
+    if (correctAnswer === userAnswer) {
+      document.getElementById("question-element").remove();
+      currentIndex++;
+      renderQuestion();
+      countdownClock += 5;
+    } else {
+      document.getElementById("question-element").remove();
+      currentIndex++;
+      countdownClock -= 5;
+      renderQuestion();
+    }
+  } else {
+    renderScore();
+  }
+};
+
+// Start Quiz - remove elements, start timer and render question
+const startQuiz = function () {
+  startQuizDiv.remove();
+
+  renderTimer();
+
+  startTimer();
+
+  renderQuestion();
+};
+
+// Render Score: Remove last question, construct score container and allow user to submit initials
+const renderScore = function () {
+  document.querySelector("#clock").remove();
+  document.getElementById("question-element").remove();
+  const finalScore = countdownClock;
+
+  const scoreDiv = document.createElement("div");
+  scoreDiv.setAttribute("class", "score-div");
+  scoreDiv.setAttribute("id", "score-div");
+
+  const scoreContainer = document.createElement("div");
+  scoreContainer.setAttribute("id", "score-container");
+  scoreContainer.setAttribute("class", "score-container");
+
+  const scoreHeader = document.createElement("h2");
+  scoreHeader.setAttribute("id", "score-title");
+  scoreHeader.textContent = `Your Score Is: ${countdownClock}`;
+
+  const enterInitials = document.createElement("p");
+  enterInitials.setAttribute("class", "user-initials");
+  enterInitials.textContent = `Enter Initials:`;
+
+  const submitButton = document.createElement("button");
+  submitButton.setAttribute("id", "submit-btn");
+  submitButton.setAttribute("class", "submit-btn");
+  submitButton.textContent = `Submit Score`;
+
+  const userInitials = document.createElement("input");
+  userInitials.setAttribute("id", "user-input");
+  userInitials.setAttribute("class", "user-input");
+
+  scoreContainer.appendChild(scoreDiv);
+  scoreContainer.appendChild(scoreHeader);
+  scoreContainer.appendChild(enterInitials);
+  scoreContainer.appendChild(userInitials);
+  scoreContainer.appendChild(submitButton);
+  scoreContainer.addEventListener("click", answerValidation);
```

Comment:
This line is causing console errors when trying to submit the form, as you're trying to run the `answerValidation` function whenever we click inside the container. Is this an error from a copy and paste that shouldn't be there?